### PR TITLE
Fix #2628 - allow meta generator to be diabled

### LINKED ISSUE #2628: Add option to disable meta generator tag
See #2620.

### PR DESCRIPTION
**Pull request by adarshdec23**

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -11,6 +11,8 @@ Features
 * Sort posts in the following order (most important last): source path
   (A-Z), title (A-Z), date (reverse chronological order), priority
   meta number (descending). (Issue #2627)
+* Add META_GENERATOR_TAG option in conf.py allowing the meta generator 
+  to be disabled if needed. (Issue #2628)
```
Comment (felixfontein) [inline CHANGES.txt]: You need to add this more at the top, before the changelist for version v7.8.3 starts. Now it looks like this was added in v7.8.3, while it wasn't.

Just start an empty `Features` section at the top of the file and move the comment there.

Comment (Kwpolska) [inline CHANGES.txt]: @felixfontein This should be the entire New in master/Features/Bugfixes template. I’ll take care of it myself, it’s a pretty minor thing.

**Comment by felixfontein** [comment]
Sure, fine for me :)

**Comment by felixfontein** [comment]
Thanks @adarshdec23 for implementing this!

**Comment by Kwpolska** [comment]
Thanks for contributing to Nikola, @adarshdec23!

**Comment by adarshdec23** [comment]
Thanks for the merge! :-)

**Pull request merged**